.github/workflows/sphinx_asdf_ci.yml

```diff
@@ -0,0 +1,150 @@
+name: Sphinx ASDF CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    branches:
+
+jobs:
+  tox:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - name: Python 3.9
+            python-version: 3.9
+            os: ubuntu-latest
+            script: |
+              python -m pip install --upgrade pip
+              python -m pip install .[tests]
+              python -m pip install sphinx
+            toxenv: py39
+
+          - name: Python 3.8
+            python-version: 3.8
+            os: ubuntu-latest
+            script: |
+              python -m pip install --upgrade pip
+              python -m pip install .[tests]
+              python -m pip install sphinx
+            toxenv: py38
+
+          - name: Python 3.7
+            python-version: 3.7
+            os: ubuntu-latest
+            script: |
+              python -m pip install --upgrade pip
+              python -m pip install .[tests]
+              python -m pip install sphinx
+            toxenv: py37
+
+          - name: Python 3.6
+            python-version: 3.6
+            os: ubuntu-latest
+            script: |
+              python -m pip install --upgrade pip
+              python -m pip install .[tests]
+              python -m pip install sphinx
```

Comment:
`sphinx` should be a dependency perhaps?  Or certainly a `[tests]` extra dependency.   So if not there, add it, and remove all references to needing it installed here.